### DISCUSSION TITLE
Improve Definition Sources

### DISCUSSION THREAD

**Discussion by mark-jensen** [CommonCoreOntology/CommonCoreOntologies#370] [Announcements]
CCO currently has over 500 terms that link to Wikipedia as their definition source. This has increasingly become an issue of concern for some users. One problem is that Wikipedia is constantly changing. Use of permalink somewhat helps but does not alleviate concerns of the stability of Wikipedia as a trustworthy source. Another concern is the ostensible authority of Wikipedia vs. other sources, as well as the monotone presence of its use in CCO.

Propose that we begin updating and improving definition sources in CCO on two axes.
1. replace Wikipedia as the source wherever possible with one or more authoritative source.
2. add sources to terms, esp. high level ones, that don't currently have them.

Reaching out to volunteers to help. Using this issue to create a branch.

**Comment by cameronmore** [comment]
Already volunteered.

We should go through this ontology-by-ontology and perhaps hit key terms in some ontologies to make little progress everywhere before making systematic progress module by module.


There are a few questions to answer before committing anything:
To think through the priority and quality of sources. For example, should we prioritize a Merriam Webster source or Oxford English Dictionary Source? Further, we need to decide how to cite these sources. For online resources, a URI seems appropriate, but should we use dc terms bibliographic citation for articles and books? Moreover, what citation style should we use? (APA or MLA is common, I personally prefer Chicago). Also, should the value of the annotation be, strictly a URI or citation or looser like "Adapted from [insert source]" or "Quotes from [insert source]". Or, a case by case basis.

**Comment by mark-jensen** [comment]
Thanks @cameronmore. I auto-generated an issue-specific branch. See sidebar on right under "Development". Please use that one. 

Agree with your strategy. We will have to discuss specifics of the implementation. Hopefully that won't delay getting started.

**Comment by neilotte** [comment]
@mark-jensen @cameronmore Moving this into the Discussion session. I recommend we consolidate this under the styleguide discussion.
